client/a8c-for-agencies/sections/sites/site-preview-pane/site-preview-pane-content/style.scss

```diff
@@ -112,4 +112,8 @@
 	.site-expanded-content__card-content-score-title {
 		text-align: left;
 	}
+
+	a.site-expanded-content__card-button {
+		color: var(--studio-automattic-90);
```

Comment:
`--color-surface-backdrop`

Note that the `--studio-automattic-90` does not exist

Reply:
Changed them all, thanks!